packages/api-plugin-fulfillment-method-pickup-store/src/util/validateOrderMethodsstore.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @summary Sample dummy function validating the data requirements for this metod
+ * @param {Object} context - Context object
+ * @param {Object} commonOrder - Current order which provide available details to perform validation
+ * @param {Object[]} validationResults - Validation results collected till now
+ * @returns {Object[]} validationResults - with the validation details populated
+ */
+export default function validateOrderMethodsstore(context, commonOrder, validationResults = []) {
```

Comment:
Capitalize the `store` in the function name, as well as in the file name.
```suggestion
export default function validateOrderMethodsStore(context, commonOrder, validationResults = []) {
```